ouroboros-network/src/Ouroboros/Network/PeerSelection/LedgerPeers.hs

```diff
@@ -57,6 +57,10 @@ isLedgerPeersEnabled :: UseLedgerAfter -> Bool
 isLedgerPeersEnabled DontUseLedger = False
 isLedgerPeersEnabled _             = True
 
+-- | Identifies a peer as coming from ledger or not
+data LedgerPeer = IsLedgerPeer | IsNotLedgerPeer
+  deriving (Eq, Show)
+
```

Comment:
What about:
```suggestion
-- | Identifies a peer as coming from ledger or not
data IsLedgerPeer = LedgerPeer | NotLedgerPeer
  deriving (Eq, Show)

```

Reply:
I prefer the affirmative for boolean like data types, it is more clear to me. With your suggestion I'd only be sure if I'd see `NoLedgerPeer`

Reply:
You're right, probably this most constistent way, isn't it?
```
data IsLedgerPeer = IsLedgerPeer | IsNotLedgerPeer
  deriving (Eq, Show)
```

Reply:
Yes!

Reply:
This way when one sees `IsLedgerPeer` type one expects an proof of it as a term, while `LedgerPeer` suggest the term is a ledger peer.